toolchain/parser/parse_tree.cpp

```diff
@@ -88,16 +88,39 @@ auto ParseTree::GetNodeText(Node n) const -> llvm::StringRef {
   return tokens_->GetTokenText(node_impls_[n.index_].token);
 }
 
+auto ParseTree::PrintNode(llvm::raw_ostream& output, Node n, int depth,
+                          bool preorder) const -> bool {
+  const auto& n_impl = node_impls_[n.index()];
+  output.indent(2 * depth);
+  output << "{";
+  // If children are being added, include node_index in order to disambiguate
+  // nodes.
+  if (preorder) {
+    output << "node_index : " << n.index_ << ", ";
```

Comment:
```suggestion
    output << "node_index: " << n.index_ << ", ";
```

I think an extra space snuck in?